static/api-docs/2024.1/Python-API/_modules/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html class="writer-html5" lang="en" >
+<head>
+  <meta charset="utf-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Overview: module code &mdash; CloudShell Python API 2022.3.0.185573 documentation</title>

```

Comment:
can you check this? seems like the python API packages is generated with 22.3 version (it appears in multiple places)

Reply:
yeah